lib/taskana-core-test/src/test/java/acceptance/task/requestchanges/RequestChangesAccTest.java

```diff
@@ -0,0 +1,183 @@
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.api.function.ThrowingConsumer;
+
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.api.models.ClassificationSummary;
+import pro.taskana.common.internal.util.EnumUtil;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.TaskState;
+import pro.taskana.task.api.exceptions.InvalidOwnerException;
+import pro.taskana.task.api.exceptions.InvalidTaskStateException;
+import pro.taskana.task.api.models.ObjectReference;
+import pro.taskana.task.api.models.Task;
+import pro.taskana.testapi.DefaultTestEntities;
+import pro.taskana.testapi.TaskanaInject;
+import pro.taskana.testapi.TaskanaIntegrationTest;
+import pro.taskana.testapi.builder.TaskBuilder;
+import pro.taskana.testapi.builder.WorkbasketAccessItemBuilder;
+import pro.taskana.testapi.security.WithAccessId;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.exceptions.MismatchedWorkbasketPermissionException;
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+
+@TaskanaIntegrationTest
+class RequestChangesAccTest {
+  @TaskanaInject TaskService taskService;
+
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup(ClassificationService classificationService, WorkbasketService workbasketService)
+      throws Exception {
+    defaultClassificationSummary =
+        defaultTestClassification().buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary = defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+
+    defaultObjectReference = DefaultTestEntities.defaultTestObjectReference().build();
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_RequestChanges_When_TaskIsClaimed() throws Exception {
+    Instant now = Instant.now();
+    Task task = createTaskInReviewByUser("user-1-1").buildAndStore(taskService);
+
+    Task result = taskService.requestChanges(task.getId());
+
+    assertThat(result.getState()).isEqualTo(TaskState.READY);
+    assertThat(result.getOwner()).isNull();
+    assertThat(result.getModified()).isAfterOrEqualTo(now);
+  }
```

Comment:
I am missing this exact test with `forceRequestChanges`.
Same applies to the RequestReviewAccTest from TSK-1915, where this test is also missing. Do you mind adding that one too?